apps/web_delivery/web/views/error_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule WebDelivery.ErrorView do
+  use WebDelivery.Web, :view
+
+  def render("404.json", _assigns), do: "Not found"
+
+  def render("500.json", _assigns), do: "Internal server error"
```

Comment:
👍 It’s a matter of preference. I went with the string sigil as you suggested since it is probably more readable.